src/docs/development/tools/devtools/visual-debugging.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Visual debugging
+description: A guide to Flutter's visual debugging functionality.
+---
+
+The Flutter Inspector provides several options for visually debugging your app.
+These are the options available from the inspector within the DevTools.
+
+## Slow Animations
+
+When enabled, this option runs animations 5 times slower for easier visual
+inspection.
+This can be useful if you want to carefully observe and tweak an animation that
+doesn’t look quite right.
+
+This can also be set in code:
+
+```dart
+import 'package:flutter/scheduler.dart' 
+
+timeDilation = 5.0;
+```
+ 
+This will slow the animations by 5x.
+
+### See also
+
+* [Flutter documentation: timeDilation property](https://api.flutter.dev/flutter/scheduler/timeDilation.html) 
+
+### Example
+
+![Screen recording showing normal animation speed]({% asset tools/devtools/debug-toggle-slow-animations-disabled.gif @path %})
+![Screen recording showing slowed animation speed]({% asset tools/devtools/debug-toggle-slow-animations-enabled.gif @path %})
+
+## Show Guidelines
+
+This option draws guidelines on your app which show render boxes, alignments,
+paddings, scroll views, clippings and spacers.
+
+This tool can be used for better understanding your layout, for instance
+finding unwanted paddings or understanding how widgets are being aligned.
+
+You can also enable this in code:
+
+```dart
+debugPaintSizeEnabled = true;
+```
+
+### Render Boxes
+
+Widgets that draw to the screen create a [render box][]; they’re the 
+building blocks of Flutter layouts. They’re shown with a bright blue border:
+
+![Screenshot of render box guidelines]({% asset tools/devtools/debug-toggle-guideline-render-box.png @path %})
+
+
+### Alignments
+
+Alignments are shown with yellow arrows. These arrows show the vertical
+and horizontal offsets of a widget relative to its parent.
+For example this button’s icon is shown as being centered by the four arrows:
+
+![Screenshot of alignment guidelines]({% asset tools/devtools/debug-toggle-guidelines-alignment.png @path %})
+
+### Paddings
+
+Paddings are shown with a semi-transparent blue background:
+
+![Screenshot of padding guidelines]({% asset tools/devtools/debug-toggle-guidelines-padding.png @path %})
+
+### Scroll views
+
+Widgets with scrolling contents such as list views are shown with green arrows:
+
+![Screenshot of scroll view guidelines]({% asset tools/devtools/debug-toggle-guidelines-scroll.png @path %})
+
+### Clipping
+
+Clippings, such as when using the [ClipRect widget][], are shown
+with a dashed pink line with a scissors icon:
+
+[ClipRect widget]: https://api.flutter.dev/flutter/widgets/ClipRect-class.html
```

Comment:
This is a global change. Use the macros for URLs when available. (Defined in _config.yml at the top of the website repo.)

```suggestion
[ClipRect widget]: {{site.api}}/flutter/widgets/ClipRect-class.html
```